t/01-basic.t

```diff
@@ -89,17 +90,6 @@ is_deeply(
     'Baz::test_apply returns expected list'
 );
 
-is(
-    ( List::AllUtils::first { $_ > 5 } ( 1, 2, 5, 22, 7 ) ),
```

Comment:
Aaaah, so that's why. Makes sense.

I don't think we can use `AUTOLOAD` here, as perl must know the functions' prototype beforehand do it won't panic seeing things like `any { ... } @blah`.  But I have a plan B. There will be an update on this PR somewhen after breakfast. :-)